.github/ISSUE_TEMPLATE/doc-issue.md

```diff
@@ -4,15 +4,47 @@ about: Create an issue to help us improve
 
 ---
 
-**Before you enter an issue**
+## Before you enter an issue
 
-- If the issue is a simple typo or similar correction, you can just submit a PR. See [the contributor guide](https://docs.microsoft.com/contribute/#quick-edits-to-existing-documents) for instructions, if needed.
-- If you are are struggling with a tutorial, check your code against the completed sample for differences.
-- If the issue is a general support question rather than a docs issue, consider a forum site instead of a docs issue.
-- If the issue is relating to site design, create an issue at [MicrosoftDocs/Feedback](https://github.com/MicrosoftDocs/Feedback/Issues/new/choose)
+If the issue is:
 
-**Include version and OS information**
-1. Include the OS you're using.
-1. Include the version of .NET you're using. Include "Framework" or "Core" as well.
+- A simple typo or similar correction, you can submit a PR. See [the contributor guide](https://docs.microsoft.com/contribute/#quick-edits-to-existing-documents) for instructions, if needed.
+- A general support question rather than a docs issue, consider a forum site instead of a docs issue.
+- Related to site design, create an issue at [MicrosoftDocs/Feedback](https://github.com/MicrosoftDocs/Feedback/Issues/new/choose).
 
-Be as descriptive as you can about the issue.
+If you're struggling with a tutorial, compare your code with the completed sample.
+
+## Environment details
+
+To expedite the triage process, be as descriptive as possible.
+
+### Operating system
+
+Check the OS being used, and include a version number. For Linux, indicate the distribution name (for example, CentOS).
+
+- [ ] Linux
+- [ ] macOS
+- [ ] Windows
+
+### Editor / IDE
+
+Check the editor or IDE being used, and include a version number.
+
+- [ ] Visual Studio
+- [ ] Visual Studio Code
+- [ ] Visual Studio for Mac
+- [ ] Other (specify)
+
+### Target framework
+
+Check the .NET target framework(s) being used, and include the version number(s).
+
+- [ ] .NET Core
+- [ ] .NET Framework
+- [ ] .NET Standard
+
+If using the .NET Core SDK, include the output from the `dotnet --info` command:
```

Comment:
Not optional. Few know they are using the SDK when that's what the problem is. Don't make it optional.

Reply:
How about having someone paste the output from Visual Studio's About - Version command if they are using the framework?

Reply:
I like that idea!